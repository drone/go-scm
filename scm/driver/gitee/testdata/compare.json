{
  "base_commit": {
    "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/e3c0ff4d5cef439ea11b30866fb1ed79b420801d",
    "sha": "e3c0ff4d5cef439ea11b30866fb1ed79b420801d",
    "html_url": "https://gitee.com/kit101/drone-yml-test/commit/e3c0ff4d5cef439ea11b30866fb1ed79b420801d",
    "comments_url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/e3c0ff4d5cef439ea11b30866fb1ed79b420801d/comments",
    "commit": {
      "author": {
        "name": "kit101",
        "date": "2021-08-23T23:45:53+08:00",
        "email": "qkssk1711@163.com"
      },
      "committer": {
        "name": "kit101",
        "date": "2021-08-23T23:45:53+08:00",
        "email": "qkssk1711@163.com"
      },
      "message": "add/update/delete/rename\n",
      "tree": {
        "sha": "55d4c682f3e2b276cc40c70e59cebb17dcb1273a",
        "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/git/trees/55d4c682f3e2b276cc40c70e59cebb17dcb1273a"
      }
    },
    "author": {
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "url": "https://gitee.com/api/v5/users/kit101",
      "html_url": "https://gitee.com/kit101",
      "remark": "",
      "followers_url": "https://gitee.com/api/v5/users/kit101/followers",
      "following_url": "https://gitee.com/api/v5/users/kit101/following_url{/other_user}",
      "gists_url": "https://gitee.com/api/v5/users/kit101/gists{/gist_id}",
      "starred_url": "https://gitee.com/api/v5/users/kit101/starred{/owner}{/repo}",
      "subscriptions_url": "https://gitee.com/api/v5/users/kit101/subscriptions",
      "organizations_url": "https://gitee.com/api/v5/users/kit101/orgs",
      "repos_url": "https://gitee.com/api/v5/users/kit101/repos",
      "events_url": "https://gitee.com/api/v5/users/kit101/events{/privacy}",
      "received_events_url": "https://gitee.com/api/v5/users/kit101/received_events",
      "type": "User"
    },
    "committer": {
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "url": "https://gitee.com/api/v5/users/kit101",
      "html_url": "https://gitee.com/kit101",
      "remark": "",
      "followers_url": "https://gitee.com/api/v5/users/kit101/followers",
      "following_url": "https://gitee.com/api/v5/users/kit101/following_url{/other_user}",
      "gists_url": "https://gitee.com/api/v5/users/kit101/gists{/gist_id}",
      "starred_url": "https://gitee.com/api/v5/users/kit101/starred{/owner}{/repo}",
      "subscriptions_url": "https://gitee.com/api/v5/users/kit101/subscriptions",
      "organizations_url": "https://gitee.com/api/v5/users/kit101/orgs",
      "repos_url": "https://gitee.com/api/v5/users/kit101/repos",
      "events_url": "https://gitee.com/api/v5/users/kit101/events{/privacy}",
      "received_events_url": "https://gitee.com/api/v5/users/kit101/received_events",
      "type": "User"
    },
    "parents": [
      {
        "sha": "537575f44a09c57dfc472e26fe067754fd2f9374",
        "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/537575f44a09c57dfc472e26fe067754fd2f9374"
      }
    ]
  },
  "merge_base_commit": {
    "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/e3c0ff4d5cef439ea11b30866fb1ed79b420801d",
    "sha": "e3c0ff4d5cef439ea11b30866fb1ed79b420801d",
    "html_url": "https://gitee.com/kit101/drone-yml-test/commit/e3c0ff4d5cef439ea11b30866fb1ed79b420801d",
    "comments_url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/e3c0ff4d5cef439ea11b30866fb1ed79b420801d/comments",
    "commit": {
      "author": {
        "name": "kit101",
        "date": "2021-08-23T23:45:53+08:00",
        "email": "qkssk1711@163.com"
      },
      "committer": {
        "name": "kit101",
        "date": "2021-08-23T23:45:53+08:00",
        "email": "qkssk1711@163.com"
      },
      "message": "add/update/delete/rename\n",
      "tree": {
        "sha": "55d4c682f3e2b276cc40c70e59cebb17dcb1273a",
        "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/git/trees/55d4c682f3e2b276cc40c70e59cebb17dcb1273a"
      }
    },
    "author": {
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "url": "https://gitee.com/api/v5/users/kit101",
      "html_url": "https://gitee.com/kit101",
      "remark": "",
      "followers_url": "https://gitee.com/api/v5/users/kit101/followers",
      "following_url": "https://gitee.com/api/v5/users/kit101/following_url{/other_user}",
      "gists_url": "https://gitee.com/api/v5/users/kit101/gists{/gist_id}",
      "starred_url": "https://gitee.com/api/v5/users/kit101/starred{/owner}{/repo}",
      "subscriptions_url": "https://gitee.com/api/v5/users/kit101/subscriptions",
      "organizations_url": "https://gitee.com/api/v5/users/kit101/orgs",
      "repos_url": "https://gitee.com/api/v5/users/kit101/repos",
      "events_url": "https://gitee.com/api/v5/users/kit101/events{/privacy}",
      "received_events_url": "https://gitee.com/api/v5/users/kit101/received_events",
      "type": "User"
    },
    "committer": {
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "url": "https://gitee.com/api/v5/users/kit101",
      "html_url": "https://gitee.com/kit101",
      "remark": "",
      "followers_url": "https://gitee.com/api/v5/users/kit101/followers",
      "following_url": "https://gitee.com/api/v5/users/kit101/following_url{/other_user}",
      "gists_url": "https://gitee.com/api/v5/users/kit101/gists{/gist_id}",
      "starred_url": "https://gitee.com/api/v5/users/kit101/starred{/owner}{/repo}",
      "subscriptions_url": "https://gitee.com/api/v5/users/kit101/subscriptions",
      "organizations_url": "https://gitee.com/api/v5/users/kit101/orgs",
      "repos_url": "https://gitee.com/api/v5/users/kit101/repos",
      "events_url": "https://gitee.com/api/v5/users/kit101/events{/privacy}",
      "received_events_url": "https://gitee.com/api/v5/users/kit101/received_events",
      "type": "User"
    },
    "parents": [
      {
        "sha": "537575f44a09c57dfc472e26fe067754fd2f9374",
        "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/537575f44a09c57dfc472e26fe067754fd2f9374"
      }
    ]
  },
  "commits": [
    {
      "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/2700445cd84c08546f4d003f8aa54d2099a006b7",
      "sha": "2700445cd84c08546f4d003f8aa54d2099a006b7",
      "html_url": "https://gitee.com/kit101/drone-yml-test/commit/2700445cd84c08546f4d003f8aa54d2099a006b7",
      "comments_url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/2700445cd84c08546f4d003f8aa54d2099a006b7/comments",
      "commit": {
        "author": {
          "name": "kit101",
          "date": "2021-09-28T16:45:33+00:00",
          "email": "qkssk1711@163.com"
        },
        "committer": {
          "name": "Gitee",
          "date": "2021-09-28T16:45:33+00:00",
          "email": "noreply@gitee.com"
        },
        "message": "add feat-compare.txt.\n",
        "tree": {
          "sha": "3743833f1906b6343c0e373e9d5c5c02bb793f4d",
          "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/git/trees/3743833f1906b6343c0e373e9d5c5c02bb793f4d"
        }
      },
      "author": {
        "id": 1535738,
        "login": "kit101",
        "name": "kit101",
        "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
        "url": "https://gitee.com/api/v5/users/kit101",
        "html_url": "https://gitee.com/kit101",
        "remark": "",
        "followers_url": "https://gitee.com/api/v5/users/kit101/followers",
        "following_url": "https://gitee.com/api/v5/users/kit101/following_url{/other_user}",
        "gists_url": "https://gitee.com/api/v5/users/kit101/gists{/gist_id}",
        "starred_url": "https://gitee.com/api/v5/users/kit101/starred{/owner}{/repo}",
        "subscriptions_url": "https://gitee.com/api/v5/users/kit101/subscriptions",
        "organizations_url": "https://gitee.com/api/v5/users/kit101/orgs",
        "repos_url": "https://gitee.com/api/v5/users/kit101/repos",
        "events_url": "https://gitee.com/api/v5/users/kit101/events{/privacy}",
        "received_events_url": "https://gitee.com/api/v5/users/kit101/received_events",
        "type": "User"
      },
      "committer": null,
      "parents": [
        {
          "sha": "e3c0ff4d5cef439ea11b30866fb1ed79b420801d",
          "url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/commits/e3c0ff4d5cef439ea11b30866fb1ed79b420801d"
        }
      ]
    }
  ],
  "files": [
    {
      "sha": "b3a3b1dc93cb4323ef918aa742227356e91ddf1e",
      "filename": "feat-compare.txt",
      "status": "added",
      "additions": 1,
      "deletions": 0,
      "changes": 1,
      "blob_url": "https://gitee.com/kit101/drone-yml-test/blob/2700445cd84c08546f4d003f8aa54d2099a006b7/feat-compare.txt",
      "raw_url": "https://gitee.com/kit101/drone-yml-test/raw/2700445cd84c08546f4d003f8aa54d2099a006b7/feat-compare.txt",
      "content_url": "https://gitee.com/api/v5/repos/kit101/drone-yml-test/contents/feat-compare.txt?ref=2700445cd84c08546f4d003f8aa54d2099a006b7",
      "patch": "@@ -0,0 +1 @@\n+feat-compare\n\\ No newline at end of file\n"
    }
  ]
}