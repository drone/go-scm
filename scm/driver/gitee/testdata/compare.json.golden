[
  {
    "Path": "feat-compare.txt",
    "Added": true,
    "Renamed": false,
    "Deleted": false,
    "BlobID": "b3a3b1dc93cb4323ef918aa742227356e91ddf1e"
  }
]
